src/actions.rs

```diff
@@ -0,0 +1,886 @@
+//! Actions functionality for WebDriver.
+use crate::elements::Element;
+use std::fmt::Debug;
+use std::time::Duration;
+use webdriver::actions as WDActions;
+
+/// An action not associated with an input device (i.e. pause).
+#[derive(Debug, Clone)]
+pub enum NullAction {
+    /// Pause for the specified duration.
+    Pause {
+        /// The pause duration, given in milliseconds.
+        duration: u64,
+    },
+}
+
+impl From<NullAction> for WDActions::NullActionItem {
+    fn from(na: NullAction) -> Self {
+        match na {
+            NullAction::Pause { duration } => WDActions::NullActionItem::General(
+                WDActions::GeneralAction::Pause(WDActions::PauseAction {
+                    duration: Some(duration),
+                }),
+            ),
+        }
+    }
+}
+
+/// An action performed with a keyboard.
+#[derive(Debug, Clone)]
+pub enum KeyAction {
+    /// Pause action.
+    /// Useful for adding pauses between other key actions.
+    Pause {
+        /// The pause duration, given in milliseconds.
+        duration: u64,
+    },
+    /// Key up action.
+    Up {
+        /// The key code, e.g. `'a'`. See the [`Keys`] module for special key codes.
+        ///
+        /// [`Keys`]: crate::keys::Keys
+        value: char,
```

Comment:
Should we have
```rust
enum KeyCode {
  Special(Key),
  Type(char)
}
```
or something like it, and then use that enum type here?

Reply:
What would this buy us? `Key` converts into `char` anyway

Reply:
It would avoid the need to call `Key::into()`, but yeah, point taken :+1: